netplan_cli/cli/state.py

```diff
@@ -434,6 +434,9 @@ def __init__(self, ifname=None, all=False):
                               'Please start it.')
                 sys.exit(1)
             logging.debug('systemd-networkd.service is not active. Starting...')
+            if not utils.systemctl_is_installed('systemd-networkd.service'):
+                logging.error('systemd-networkd is required for \'netplan status\' functionality')
+                sys.exit(1)
```

Comment:
nitpick: We might want to move this above the `systemd-networkd.service is not active. Starting...` debug message, as the `systemctl start` happens only after.

Reply:
Good nitpick.  Fixed.